### PR TITLE
Prep changelog for 6.8

### PR DESCRIPTION
**Pull request by kinkie**

### CODE REVIEW

**Comment by kinkie** [comment]
Note: this PR is against v6; once accepted, it will need to be forward-ported to master

**Comment by rousskov** [comment]
Thank you for keeping these v6-specific changes to the v6 branch!

**Comment by rousskov** [comment]
> this PR is against v6; once accepted, it will need to be forward-ported to master

... but there is no rush with that second/porting step. We can even sync ChangeLog around v7 branching time!

**Review by yadij — CHANGES_REQUESTED** [review]

File: configure.ac [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_INIT([Squid Web Proxy],[6.7-VCS],[https://bugs.squid-cache.org/],[squid])
+AC_INIT([Squid Web Proxy],[6.8-VCS],[https://bugs.squid-cache.org/],[squid])
```
[the remark this answers is not in the document]
Comment (kinkie) [reply configure.ac]: reverted.

[the remark this answers is not in the document]
Comment (rousskov) [reply configure.ac]: > reverted

@kinkie, FWIW, I doubt Amos was actually requesting a reversal of AC_INIT() change, but, if, for whatever reason, you do _not_ want this PR commit to be the "formal release" commit, should not the title of this PR be updated again?

[the remark this answers is not in the document]
Comment (kinkie) [reply configure.ac]: Title updated.
I think it's safest to rollback and to stick to established procedure. I'm already testing new ground, let's do one change at a time

**Review by yadij — APPROVED** [review]

File: configure.ac (hunk 1 above)
[the remark this answers is not in the document]
Comment (yadij) [reply configure.ac]: Well, established procedure is to merge these documentation changes to master. So that line has already broken.

**Pull request merged**